pkg/agent/nodeportlocal/k8s/npl_controller.go

```diff
@@ -95,12 +96,12 @@ func NewNPLController(kubeClient clientset.Interface,
 	)
 	svcInformer.AddIndexers(
 		cache.Indexers{
-			NPLEnabledAnnotationIndex: func(obj interface{}) ([]string, error) {
+			types.NPLEnabledAnnotationIndex: func(obj interface{}) ([]string, error) {
```

Comment:
NPLEnabledAnnotationIndex is an internal constant used for building NPLController's index, it should be kept in npl_controller.go.